03/app.test.js

```diff
@@ -3,24 +3,55 @@ import randomNumber from './app';
 describe('Range', () => {
     it('Should throw exception when min not a number', () => {
         function checkNumber() {
-            randomNumber('1')
+            randomNumber('2')
         }
         expect(checkNumber).toThrow()
     })
-    
+
+    it('Should throw exception when min not a number', () => {
+        function checkNumber() {
+            randomNumber(NaN)
+        }
+        expect(checkNumber).toThrow()
+    })
+
+    it('Should inform when min is not a number', () => {
+        try {
```

Comment:
Jeśli mas warunek, a try...catch w jakimś sensie jest to powinnaś mieć `expect.assertions(1)`